src/components/ListItem.jsx

```diff
@@ -120,26 +119,46 @@ export function ListItem({
 	};
 
 	return (
-		<li className={isChecked ? checkedItemStyle : uncheckedItemStyle}>
-			<input
-				type="checkbox"
-				id={`checkbox-${id}`} // Unique identifier
-				name={name}
-				onChange={handleChecked}
-				checked={isChecked}
-			></input>
-			<label className="flex px-2 items-center w-full h-full text-lg xsm:text-xs sm:text-sm">
-				{capitalizeFirstLetterOfEachWord(name)}
-			</label>
-			<span
-				className={`px-2 py-1 mx-2 rounded-lg min-w-fit`}
-				style={{ backgroundColor: tagColor }}
+		<li>
+			<div
+				htmlFor={`checkbox-${id}`}
+				onClick={handleChecked}
+				className={`cursor-pointer bg-white shadow hover:shadow-md h-[72px] flex items-center justify-between rounded-lg p-6 transition-shadow duration-300 ease-in-out`}
+				onKeyPress={handleKeyPress}
+				role="button"
+				tabIndex="0"
 			>
-				{!isChecked ? urgency : 'checked'}
-			</span>
-			<button onClick={handleDelete} className="delete-button">
-				<TrashIcon />
-			</button>
+				<div className="flex gap-4 text-base">
+					<div className="relative flex items-center">
+						<input
+							type="checkbox"
+							id={`checkbox-${id}`}
+							name={name}
+							onChange={handleChecked}
+							checked={isChecked}
+							className="h-4 w-4 checked:bg-tcl-blue checked:border-0 outline-none bg-gray-50 cursor-pointer appearance-none border border-gray-300 rounded-[4px]"
+							tabIndex={-1}
+						/>
+						<div className="absolute top-[12px] left-1/2 transform -translate-x-1/2 -translate-y-1/2">
+							<IoIosCheckmark className="text-white" size={22} />
```

Comment:
It looks like the white checkmark shows up all the time, regardless of whether the item is checked. It's just very subtly there.